### PR TITLE
Updating upgrade your widget guide

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:

The PAT team identified a number of updates required for the [Upgrade your widget](https://developer.okta.com/docs/guides/oie-upgrade-sign-in-widget/main/) article.
- **Preview URL** https://62de9d16b50ddd22e14c6b8c--reverent-murdock-829d24.netlify.app/docs/guides/oie-upgrade-sign-in-widget/main/
- **What's changed?** This PR:
  * Replaces all the screenshots of code with actual code, which is much better in terms of maintenance, readibility, and accessibility
  * Removes the `parseSchema()` method from the `registration` object listing, as this is no longer supported
  * Updates the `idpDiscovery` declaration code block as shown in the JIRA
  * Updates the "OpenID Connect/social authentication" section as shown in the JIRA

- Outstanding questions for reviewer:
  * I kept the `registration: true` declaration inside the `features` object, as the text is telling the reader to remove it. Is that OK?
  * In the JIRA, it mentions the "Bootstrapping from a recovery token" section and says "This should link to the new method of pay-loading the recovery token (with state)… “Replaced with process XYZ”". I'm not sure what I should do here. What URL should I think to, and what wording should I use?
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-516851](https://oktainc.atlassian.net/browse/OKTA-516851)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 1]
```diff
@@ -101,45 +101,92 @@ var config = {
...
-
-You no longer need the `idps` JavaScript object and `className` property in the widget, and you can remove them as follows.
+You no longer require the `idps` JavaScript object and `className` property in the widget (shown below for reference), and you can remove them.  This is optional as the Okta Identity Engine will now include IdPs based on the routing rules.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:143]:
```suggestion
You no longer require the `idps` JavaScript object and `className` property in the widget. You can remove the `idps` definition that is shown in the following snippet. 
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 2]
```diff
@@ -101,45 +101,92 @@ var config = {
...
 
-![Displays the IdPs object and className property in JavaScript to remove](/img/SIW_Upgrade_Config_Change3.png)
+```JavaScript
+idps: [
+  { type: 'GOOGLE', id: '0oagjkh' },
+  { type: 'FACEBOOK', id: '0oagjkh' },
+]
+```
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:150]: Add this after the snippet?

The widget displays IdPs based on Identity Engine routing rules.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 3]
```diff
@@ -101,45 +101,92 @@ var config = {
...
 
 ### Smart card IdP
 
-You no longer need to configure the authentication settings for the [smart card IdP](https://help.okta.com/okta_help.htm?id=ext-idp-smart-card-workflow), as it's no longer supported. You can remove the authentication settings for the smart card IdP (`piv`) as follows.
+You no longer need to configure the authentication settings for the [smart card IdP](https://help.okta.com/okta_help.htm?id=ext-idp-smart-card-workflow), as it's no longer supported.
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:154]:
```suggestion
[Smart card IdP](https://help.okta.com/okta_help.htm?id=ext-idp-smart-card-workflow) is no longer supported.
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 4]
```diff
@@ -101,45 +101,92 @@ var config = {
...
+
+Remove the authentication settings for the smart card IdP (`piv`) shown below:
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:156]:
```suggestion
Remove the authentication settings for the smart card IdP (`piv`):
```

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md [hunk 5]
```diff
@@ -101,45 +101,92 @@ var config = {
...
 
 The recovery token is dynamic and is automatically passed into the initialization of the widget. A value in the `recoveryToken` setting currently doesn't have any effect on widget function, though, the setting takes effect in the future.
 
 ### Okta dashboard or custom dashboard sign-in flow
 
-For an Okta dashboard sign-in, you no longer need to configure a redirect to the Okta Identity Cloud, create an Okta session, and then open a URL specified in the widget. You can remove the redirect configuration as follows.
+For an Okta dashboard sign-in, you no longer need to configure a redirect to the Okta Identity Cloud, create an Okta session, and then open a URL specified in the widget.
 
-![Displays the Okta dashboard sign-in](/img/SIW_Upgrade_Config_Change6.png)
+Remove the redirect configuration (`setCookieAndRedirect()`) line shown below:
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:181]:
```suggestion
Remove the redirect configuration (`setCookieAndRedirect()`) line, shown in the following snippet:
```

**Review by vanngo-okta — APPROVED** [review]
Some style nits. Approving to not hold you up. 👍

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md (hunk 1 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:143]: Agreed with your wish to change this, but I did something slightly different, which I'll send up here in the next commit.

File: packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md (hunk 2 above)
Comment (chrisdmills-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-upgrade-sign-in-widget/main/index.md:150]: Again, pretty much did what you suggested just slightly differently. Really appreciate the idea.

**Pull request merged**